ERCS/erc-draft_bound_signatures.md

```diff
@@ -0,0 +1,72 @@
+---
```

Comment:
```suggestion
---
eip: 8111
```
Assigning next sequential EIP/ERC/RIP number.
Numbers are assigned by editors & associates.

Please also update the filename.